src/constants/skills.js

```diff
@@ -247,17 +247,16 @@ const icons = {
     "https://raw.githubusercontent.com/devicons/devicon/master/icons/csharp/csharp-original.svg",
   d3js:
     "https://raw.githubusercontent.com/devicons/devicon/master/icons/d3js/d3js-original.svg",
-  django:
-    "https://raw.githubusercontent.com/devicons/devicon/master/icons/django/django-original.svg",
+  django: "https://miro.medium.com/max/1200/1*1OBwwxzJksMv0YDD-XmyBw.png",
   docker:
     "https://raw.githubusercontent.com/devicons/devicon/master/icons/docker/docker-original-wordmark.svg",
   dotnet:
     "https://raw.githubusercontent.com/devicons/devicon/master/icons/dot-net/dot-net-original-wordmark.svg",
   electron:
     "https://raw.githubusercontent.com/devicons/devicon/master/icons/electron/electron-original.svg",
-  express:
-    "https://raw.githubusercontent.com/devicons/devicon/master/icons/express/express-original-wordmark.svg",
-  go: "https://raw.githubusercontent.com/devicons/devicon/master/icons/go/go-original.svg",
+  express: "https://www.mementotech.in/assets/images/icons/express.png",
+  go:
```

Comment:
Why do some of them get 2 lines?

Reply:
This is because of my code formatter if you request I will remove these lines.